lib/widgets/core/text_prompt.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class TextPrompt extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(
+            'Ready to start?',
```

Comment:
Can you mark these strings as translatable? Just add them to app_en.arb and then you can access the key with `AppLocalizations.of(context).yourKey`

Reply:
(the generated files are created on every build but sometimes the IDE still can't recognize them and mark the AppLocalizations key, but if it builds, just ignore the error)

Reply:
I have marked the strings as translatable and commited the changes.

Reply:
thanks, looks good! Will run some tests later today and then we can merge this :)